documentation/features/context-menu.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebarTitle: Context Menu
+title: Context Menu
+description:
+  Use context-aware menus to assist with your actions and improve your workflow.
+---
+
+Context-aware menus enable quick actions like creating variables or adding query
+parameters by selecting text or elements. The context menu offers three options:
+**Set as Variable**, **Add to Parameters**, and **Open Request in New Tab**.
+
+<Info>
+  The "Add to Parameters" option is available only when using query parameters.
+</Info>
+
+## Set as Variable
+
+> You can use the **Set as Variable** option from the context menu to assign a
+> selected value to an environment variable.
+
+> Hoppscotch supports various variable scopes, including global, environment,
+> and request-specific variables. For more details on these scopes
+> [click here](/documentation/features/variables).
+
+### How to use "Set as Variable"
+
+After setting an [environment variable](/documentation/features/environments),
+you can reference it using the format `<<variable name>>`. This allows you to
+store For example, `<<baseURL>>` can be used to dynamically insert the base URL
+across your requests.
+
+## Add to Parameters
+
+The **Add to Parameters** option allows you to quickly add parameters along with
+their values.
+
+When you select **Add to Parameters**, it automatically captures the parameter
+and its value. For example, if the `baseURL` is
+`https://jsonplaceholder.typicode.com/?user=1`, now the **user** with ID **1**
```

Comment:
```suggestion
`https://jsonplaceholder.typicode.com/?user=1`, now the `user` with ID `1`
```

Reply:
Can you tag me once these changes are done?

Reply:
Sure!

Reply:
@SanskritiHarmukh review changes!